src/components/map/MapService.js

```diff
@@ -548,6 +561,38 @@ goog.require('ga_urlutils_service');
                 attributionUrl: 'https://www.swisstopo.admin.ch/' + lang +
                     '/home.html'
               };
+
+              // 3D Tileset
+              var tileset3d = [
+                'ch.swisstopo.swisstlm3d.3d',
+                'ch.swisstopo.bridgestest.3d',
+                'ch.swisstopo.swissnames3d.3d'
+              ];
+              var tilesetTs = [
+                '20161217',
+                '20161121',
+                '20160909'
+              ];
+
+              var params = gaPermalink.getParams();
+              var pTileset3d = params['tileset3d'];
+              var pTilesetTs = params['tilesetTs'];
+
+              tileset3d = pTileset3d ? pTileset3d.split(',') : tileset3d;
+              tilesetTs = pTilesetTs ? pTilesetTs.split(',') : tilesetTs;
+
+              tileset3d.forEach(function(tileset3dId, idx) {
+                if (tileset3dId) {
+                  response.data[tileset3dId] = {
+                    type: 'tileset3d',
+                    serverLayerName: tileset3dId,
+                    timestamps: [tilesetTs[idx]],
+                    attribution: 'swisstopo',
+                    attributionUrl: 'https://www.swisstopo.admin.ch/' + lang +
+                        '/home.html'
+                  };
+                }
+              });
```

Comment:
As far as I know no. But we should before the official release of 3d.

There is also others hardcoded layers thing in this file we could move to the service or db